sdk/lib/rtl/memstream.c

```diff
@@ -205,6 +205,7 @@ RtlReadOutOfProcessMemoryStream(
     ULONG CopyLength;
     PRTL_MEMORY_STREAM Stream = IStream_To_RTL_MEMORY_STREAM(This);
     SIZE_T Available = (PUCHAR)Stream->End - (PUCHAR)Stream->Current;
+    SIZE_T nBytesRead = 0;
```

Comment:
Not a good variable name. We don't use Hungarian in Rtl, and it's not really distinct from BytesRead. Perhaps LocalBytesRead or MemoryBytesRead?

Reply:
Changed it to LocalBytesRead and also fixed the unconditional dereference of BytesRead, which is an _Out_opt_ parameter.